research/src/pages/tabs.proposal.mdx

```diff
@@ -14,7 +14,22 @@ showInMenu: false
 
 ### Background <a href="#background" id="background"></a>
```

Comment:
You're the first one to have a full background on this. I had even asked in the template at one point what the reason for this section was and decided to not push too hard on it. I like this content - not sure if it should lead the spec however. I think near the end this makes more sense as I think the primary reason folks will come here is to look for implementation information but I think the additional WIKI type content is useful. If you're able to reduce it some I think that would be helpful as well but keep the spirit.

Reply:
I rewrote this, much shorter, but I still don't think it is what you want.